index.html

```diff
@@ -2215,11 +2215,30 @@ <h3>Verification Relationship Validation</h3>
       <section>
         <h3>Canonicalization Method Correctness</h3>
 
-        <p class="issue">
-Canonicalization mechanisms utilized for normalizing input to hashing functions
-need to have vetted mathematical proofs associted with them. Canonicalization
-mechanisms that create collisions in hash functions can be used to attack
-digital signatures.
+        <p>
+One of the algorithmic processes that is used by this specification is
+canonicalization, which is a type of <a>transformation</a>. Canonicalization is
+the process of taking information that might be expressed in a variety of
+equivalent ways and expressing it in one way, called a "canonical form".
+        </p>
+        <p>
+The security of a resulting <a>data integrity proof</a> that utilizes
+canonicalization is highly dependent on the correctness of the algorithm. For
+example, if it is possible to confuse a canonicalization algorithm such that
+multiple different inputs produce the same output, then it's possible for one
+<a>data integrity proof</a> output to be applied to multiple inputs such that
+a verifier will not know to which input the proof applies. This can be used
+as an attack vector by adversaries.
```

Comment:
We could also draw comparisons here to pre-image / collision resistance with cryptographic hash functions. It's important that there not be more than one *semantically different* input for a given output, in the same way that it must be difficult for anyone to find more than one input for the same hash function output. This might also help address some of @TallTed's comment above.

Reply:
@TallTed would the above help?